.gitignore

```diff
@@ -88,6 +88,9 @@ venv.bak/
 .env.local
 .env.*.local
 
+# Template parameter files
+k8s/*.param
```

Comment:
What is this?

Reply:
This is the file of config parameters to build the Device Config (in OpenShift) from a script. So it doesn't get checked in :)

Reply:
ok right on! Maybe tomorrow you can tell me a bit more, i'm not sure exactly what means :)